examples/public_load_balancer/load_balancer.tf

```diff
@@ -14,9 +14,9 @@ output "load_balancer_url" {
 }
 
 resource "outscale_load_balancer_vms" "backend_vms" {
-  count              = length(outscale_vm.my_vms)
   load_balancer_name = outscale_load_balancer.my_public_lb.load_balancer_name
-  backend_vm_ids     = [outscale_vm.my_vms[count.index].vm_id]
+  backend_vm_ids     = [outscale_vm.my_vms[var.vm_countList[0]].vm_id, outscale_vm.my_vms[var.vm_countList[1]].vm_id]
```

Comment:
is using vm_countList mandatory ? var.vm_countList[0] should be equal 0 and var.vm_countList[1] should be equal to 1